ansible/deploy.py

```diff
@@ -50,6 +65,8 @@ def deploy(
     env["ANSIBLE_STDOUT_CALLBACK"] = "yaml"
     env["ANSIBLE_VAULT_IDENTITY"] = host
     env["ANSIBLE_SSH_PIPELINING"] = "true"
+    env["ANSIBLE_ROLES_PATH"] = "roles/"
```

Comment:
This actually breaks when not using WSL. Could you either remove this changes or somehow make a resilient check to only execute this statements when running inside of WSL?